package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "amd-name-resolver": "1.0.0",
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
+    "bower": "^1.8.2",
```

Comment:
I think this must have been a left over from another change? We definitely don't want to add `bower` back as a dependency...

Reply:
interesting.
must have been a bad rebase